mm2src/crypto/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+async-std = { version = "1.5", features = ["unstable"] }
```

Comment:
It seems async-std is no longer required.

Reply:
Sure, done